src/api/firebase.js

```diff
@@ -133,3 +121,61 @@ export async function matchToken(listId) {
 	const jointListTokenSnapshot = await getDocs(jointListTokenQuery);
 	return jointListTokenSnapshot;
 }
+
+export function comparePurchaseUrgency(items) {
+	const sortedItems = [];
+
+	items.forEach((item) => {
+		const differenceInDays = item.dateLastPurchased
+			? getDaysBetweenDates(item.dateLastPurchased, item.dateNextPurchased)
+			: getDaysBetweenDates(item.dateCreated, item.dateNextPurchased);
+
+		if (differenceInDays >= 60) {
```

Comment:
Another edge case to potentially consider: if the item hasn't been purchased in over 60 days (as in the dateNextPurchased was calculated to be less than 60 days from dateLastPurchased but the user hasn't purchased it in a while)